README.md

```diff
@@ -1,9 +1,8 @@
 ## GitHub Actions Build status
-|Docker Image Builds (develop) | Docker Image Builds (release) | Sonar Scanner |
+| Carma-streets-bridge | Carma-vehicle-bridge | telematics-cloud-messaging |
```

Comment:
The badges below seem to be pointing to the docker image builds for develop, release and sonar scanner - so the original text should be accurate.